src/components/results/header/style.css

```diff
@@ -65,6 +65,22 @@
 	content: "\2B07\FE0E";
 }
 
+.request-header .copy {
+	padding: 2px 3px 0;
+	font-size: 12px;
+	border-radius: 2px;
+	text-decoration: none;
+	display: inline-block;
+	vertical-align: middle;
+	margin: 2px 0 2px 8px;
+	border: 1px solid #95b7d0;
+	color: #95b7d0;
+}
+
+.request-header .copy:before {
+	content: "✂";
```

Comment:
I noticed we're using the scissors character here but that seems suggestive of the "cut" operation instead of the "copy."

We could try using U+1F4CB "clipboard" 📋